scripts/build-table.ts

```diff
@@ -36,7 +36,7 @@ async function main() {
     output = `**Total changed files:** ${fileDiffs.length}
 
 ${
-  fileDiffs.length >= threshold
+  fileDiffs.length >= +threshold
```

Comment:
There should be implicit conversion here, so there is no need to force type conversion.